tests/es/__init__.py

```diff
@@ -19,3 +19,13 @@ def elasticsearch_delete_index(index_name: str):
 def close_elasticsearch_connections(es_client):
     for conn in es_client.transport.connection_pool.connections:
         conn.pool.close()
+
+
+def clear_indexes(indexes: list, doctypes: list):
+    es_client = ElasticsearchClient.get(logger)
```

Comment:
clear_indexes should ensure it is a test index, checking that IndexSuffix.name is defined and the index name has that as a suffix.